Lesson1/Denis Sulima/task1/main.cpp

```diff
@@ -0,0 +1,85 @@
+#include <iostream>
+#include <vector>
+#include <list>
+
+std::vector<int> compoundVectors(std::vector<int>::iterator vectorFirstBegin, std::vector<int>::iterator vectorFirstEnd,
+                                std::vector<int>::iterator vectorSecondBegin, std::vector<int>::iterator vectorSecondEnd) {
+    std::vector<int>tmp {vectorFirstBegin,vectorFirstEnd};
+    tmp.insert(tmp.end(), vectorSecondBegin, vectorSecondEnd);
+    return tmp;
+}
+
+int main()
+{
+    {
+//--Task1----------------------------------------------------------------------------------------------------------------------------
+    std::cout <<"Task 1-------------------------"<<std::endl;
```

Comment:
Please write tests and export the task code to the corresponding functions without any input/output, just computing and returning the result.